README.rst

```diff
@@ -2,145 +2,108 @@
 
-What is the difference between searx and SearxNG?
-#################################################
+To get access to our Arachne Digital Slack channel, email contact[at]arachne[dot]digital with a bit about who you are and how you want to get involved.
 
-TL;DR: SearXNG is for users that want more features and bugs getting fixed quicker.
-If you prefer a minimalist software and stable experience, use searx.
+Contact us
+##########
 
-SearxNG is a fork of searx, created by a former maintainer of searx. The fork
-was created because the majority of the maintainers at the time did not find
-the new proposed features privacy respecting enough. The most significant issue is with
-engine metrics.
+You can contact us by emailing us at contact[at]arachne[dot]digital.
 
-Searx is built for privacy conscious users. It comes with a unique set of
-challenges. One of the problems we face is that users rather not report bugs,
-because they do not want to publicly share what engines they use or what search
-query triggered a problem. It is a challenge we accepted.
+If you have found any security issues with Tracery, we ask that you please contact us directly (so we can work on it without it being discovered and exploited). We will be transparent about any security issues in our documentation.
 
-The new metrics feature collects more information to make engine maintenance easier.
-We could have had better and more error reports to benefit searx maintainers.
-However, we believe that the users of searx must come first, not the
-software. We are willing to compromise on the lack of issue reports to avoid
-violating the privacy of users.
+If you have found any other bugs with Tracery, please feel free to contact us or raise an issue here in our GitHub repo.
 
-Furthermore, SearxNG is under heavy refactoring and dependencies are constantly updated, even
-if it is unnecessary. It increases the risk of introducing regressions. In searx
-we strive for stability, rather than moving fast and breaking things.
+If you have any questions or comments about Tracery, please feel free to contact us via the email address above.
 
-Is searx for me?
+Acknowledgements
 ################
 
-Are you privacy conscious user? Then yes.
+We extend our sincere appreciation to the dedicated individuals whose contributions have made Tracery a thriving open-source community and a valuable resource for the cybersecurity community!
 
-In searx we decided to double down on being privacy respecting. We are picking
-engine changes from SearxNG, but we are not implementing engine detailed
-monitoring and not adding a new UI that relies on Javascript.
+We want to extend our deepest gratitude to the original creators and maintainers of Searx, whose dedication and hard work made this powerful meta search engine possible. Without their invaluable contributions, this project would not exist.
```

Comment:
For the last sentence, I was thinking

> Without their invaluable foundations, this project would not exist.

to give more weight as - at time of writing - our fork has some slight additions to Searx but it is mostly this foundation

Reply:
Happy with this.